src/icons/general/IconAdd.js

```diff
@@ -13,17 +13,10 @@ type Props = {
 };
 
 const IconAdd = ({ className = '', color = '#000000', height = 24, title, width = 24 }: Props) => (
```

Comment:
This should also be now 32x32

Reply:
And if you dont want it to be a breaking change, you can leave the defaults to be 24x24 in the props, but still change the viewport to 32x32

Reply:
Is the 32x32 the desired size for all icons?
Should I also make the new icon 32x32 as well?

Reply:
Also, the viewport just pertains to the paint area of the icon, right? I'd need to get new assets if I wanted the viewport to be 32 x 32.

Reply:
We've been standardizing around 32x32 for anything that looks good at that aspect ratio.